crates/bevy_pbr/src/light.rs

```diff
@@ -92,9 +98,15 @@ impl Default for PointLightShadowMap {
 #[derive(Component, Debug, Clone, Copy, Reflect)]
 #[reflect(Component, Default)]
 pub struct SpotLight {
+    /// Color of the light emitted.
     pub color: Color,
+    /// Amount/strength of light emitted, in lumens.
     pub intensity: f32,
+    /// How far light extends from the surface (TODO: or center?) of the spot light.
     pub range: f32,
+    /// The size (radius) of the spot light itself.
+    ///
+    /// A radius of 0.0 (the default) is a point with no measureable size, while a radius of 1.0 is a unit sphere, etc.
```

Comment:
Radius 1.0 would give the cone cut from a unit sphere according to the outer angle.